### DISCUSSION TITLE
Is there any way to know that whether presently user has selected fixed time span or Local Clock in OpenMct web page

### DISCUSSION THREAD

**Discussion by Vaibhav2121** [nasa/openmct#4767] [Community Q&A]
Is there any way to know that whether presently user has selected fixed time span or Local Clock in OpenMct web page.

**Comment by jvigliotta** [comment]
This _is_ a bit a tough to determine. We'll probably need to add a method to the timeAPI at some point to make this easier, but in the meantime you can check if `openmct.time.clock() === undefined` then it's fixed time, if not then it's local (realtime). Clocks are only defined in local clock/realtime mode.
